docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -0,0 +1,127 @@
+developers, who are using the DAML Ledger Implementation Kit to develop a DAML
+Ledger on top of their distributed-ledger or database of choice.
+
+Use this tool to verify if the Ledger API endpoint conforms to the :doc:`DA
+Ledger Model </concepts/ledger-model/index>`.
+
+.. contents:: On this page:
+  :local:
+
+Downloading the tool
+====================
+
+TODO
+
+Extracting :code:`.dar` file required to run the tests
+======================================================
+
+Before you can run the Ledger API test tool on your ledger, you need to load a
+specific set of DAML templates onto your ledger.
+
+#. Obtain the corresponding :code:`.dar` file with:
+
+   ``ledger-api-test-tool -x``
+
+#. Load the file :code:`SemanticTests.dar` created in the current directory into your
+   Ledger.
+
+Running the tool against a custom Ledger API endpoint
+=====================================================
+
+Run this command to test your Ledger API endpoint exposed at host :code:`<host>` and
+at a port :code:`<port>`:
+
+    ``ledger-api-test-tool -h <host> -p <port>``
+
+For example
+
+    ``ledger-api-test-tool -h localhost -p 6865``
+
+Exploring options the tool provides
+===================================
+
+Run the tool with :code:`--help` flag to obtain the list of options the tool provides:
+
+   .. code-block:: console
+
+     $ ledger-api-test-tool --help
+
+     The Ledger API Test Tool is a command line tool for testing the correctness of
+     ledger implementations based on DAML and Ledger API.
+     Usage: ledger-api-test-tool [options]
+
+       --help                   prints this usage text
+       -p, --target-port <value>
+                                Ledger API server port. Defaults to 6865.
+       -h, --host <value>       Ledger API server host. Defaults to localhost.
+       --must-fail              One or more of the scenario tests must fail. Defaults to false.
+       -r, --reset              Perform a ledger reset before running the tests. Defaults to false.
+       -x, --extract            Extract the testing archive files and exit.
+
+Try out the Ledger API Test Tool against DAML Sandbox
+=====================================================
+
+To run the tool against [DAML
+Sandbox](https://docs.daml.com/tools/sandbox.html), run:
+
+   .. code-block:: console
+
+     ledger-api-test-tool -x
+     da sandbox -- SemanticTests.dar
+     ledger-api-test-tool
+
+This should always succeed! This is useful if you do not have yet a custom
+Ledger API endpoint.
+
+The DAML Sandbox starts at by default at :code:`localhost:6865`
+and the Ledger API Test Tool uses this as the default endpoint to test, hence
+hosts and ports command line arguments can be omitted.
+
+Testing your tool from continuous integration pipelines
+=======================================================
+
+To test your ledger in your CI pipelines, execute it as part of your pipeline:
```

Comment:
To test your ledger in our CI pipeline, run the `ledger-api-test` tool as part of your pipeline. You can use the `--port` option to avoid clashes of port allocations.